src/Symfony/Component/DependencyInjection/Tests/Fixtures/yaml/integration/defaults_instanceof_importance/expected.yml

```diff
@@ -16,8 +16,7 @@ services:
             # these 2 are from instanceof
             - { name: foo_tag, tag_option: from_instanceof }
             - { name: bar_tag }
-            # the tag from defaults do NOT cascade (but see #22530)
-            # - { name: from_defaults }
+            - { name: from_defaults }
```

Comment:
This is the key fix for this PR: the `from_defaults` tag is now inherited: https://github.com/symfony/symfony/blob/7010a7a76b8880c4bcf2f04f1da47e74487e2d85/src/Symfony/Component/DependencyInjection/Tests/Fixtures/yaml/integration/defaults_instanceof_importance/main.yml#L6